seonshine_frontend/src/components/molecules/searchBar/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+
+import SearchIcon from '@mui/icons-material/Search';
+import {
+  Button,
+  FormControl,
+  InputAdornment,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  Stack,
+  TextField,
+} from '@mui/material';
+
+interface SearchBarProps {
+  onSearch: (field: string, query: string) => void;
+  options: { value: string; label: string }[];
```

Comment:
Define `optionDefault` and `valueDefault`. We can pass option and value from parent to search component